docker/rpi/rpi.mk

```diff
@@ -0,0 +1,10 @@
+BOARDS += rpi
+
+local-rpi: version
+	docker buildx bake --file=docker/rpi/bake.hcl --set rpi.tagsfrigate:latest-rpi rpi
```

Comment:
`rpi.tags=`

Reply:
and I'm guessing that the `local-rpi` and `local-trt` targets should have `--load` like the `local` target does

Reply:
Thanks, forgot to add that back